### PR TITLE
[CU-23n7drb](Integration Tests): Crowdloan Rewards are now populated with our actual list of contributors

### PR DESCRIPTION
**Pull request by Dom-Roth**
Greetings, here's the update to the integration tests, to populate our actual list of contributors and test partial population.

## Issue
https://app.clickup.com/t/23n7drb
https://app.clickup.com/t/219643e

## Checklist

- [x] Populating with our actual list of contributors.
- [x] Testing partial population

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-23n7drb Test Fractional Population for Claims Pallet](https://app.clickup.com/t/23n7drb)

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-23na6t0 Composable Labs - Benchmark Guidelines](https://app.clickup.com/t/23na6t0)

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-23na6ce Composable Labs - Documentation Guidelines](https://app.clickup.com/t/23na6ce)

**Review by hussein-aitlahcen — APPROVED** [review]

File: integration-tests/runtime-tests/test/tests/crowdloanRewards/testHandlers/crowdloanHandler.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+import {sendAndWaitForSuccess, sendUnsignedAndWaitForSuccess} from "@composable/utils/polkadotjs";
+import {IKeyringPair} from "@polkadot/types/types";
+import {PalletCrowdloanRewardsModelsRemoteAccount} from "@composable/types/interfaces";
+import {u128, u32} from "@polkadot/types-codec";
+import {shares} from "@composabletests/tests/crowdloanRewards/contributions.json";
+import {expect} from "chai";
+
+
+const toHexString = bytes =>
+  Array.prototype.map.call(bytes, x => ('0' + (x & 0xFF).toString(16)).slice(-2)).join('');
```
Comment (andor0) [inline integration-tests/runtime-tests/test/tests/crowdloanRewards/testHandlers/crowdloanHandler.ts:11]: hmm... there is [toHex()](https://polkadot.js.org/docs/api/start/types.basics#everything-is-a-type).
it doesn't suit?

**Review by andor0 — APPROVED** [review]

File: integration-tests/runtime-tests/test/tests/crowdloanRewards/testHandlers/crowdloanHandler.ts (hunk 1 above)
Comment (Dom-Roth) [reply integration-tests/runtime-tests/test/tests/crowdloanRewards/testHandlers/crowdloanHandler.ts:11]: Actually don't know. This was already in the original implementation by Hussein.
I noted it down to check it for the next update.

Thanks for your input. :)

**Pull request merged**